docs/reference/packages.rst

```diff
@@ -119,7 +119,6 @@ commands which corresponds to the example above are::
 
 [TODO: Describe installation path.]
 
-
```

Comment:
Leave as-is to mimimize diff?

Reply:
I added and removed a reference tag along the way. I'll add back the line.